Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM bash:3.2
+
+# TODO: labels here. See: http://label-schema.org/rc1/
+
+ARG SHELLCHECK_VERSION=stable
+ARG SHELLCHECK_FORMAT=gcc
+
+# Install dependencies.
+RUN set -e; \
+    apk --update add \
+        git \
+        outils-sha512 \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm /var/cache/apk/*
+
+# Install shellcheck.
+RUN set -e; \
+    mkdir -p ~/stage \
+    && wget "https://storage.googleapis.com/shellcheck/shellcheck-${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" \
+    && wget "https://storage.googleapis.com/shellcheck/shellcheck-${SHELLCHECK_VERSION}.linux.x86_64.tar.xz.sha512sum" \
```

Comment:
Validating against a checksum downloaded from the exact same host doesn't offer much in terms of security (though I guess it does check file integrity)... If the point of this check is for security reasons, then we'd need to out of band copy the checksum we expect and hard-code it into the `Dockerfile`.

Reply:
It's just for file integrity, in this case. If we hard code it, we'll also be stuck with a specific version of `shellcheck`.

Reply:
k, just checking